### ISSUE TITLE
cx_oracle allows me to insert pyodbc.row object or objects in executemany() and execute. oracledb does not! 

### ISSUE THREAD

**Issue by hl1** [oracle/python-oracledb#205]
1. What versions are you using?

V.1.3.1 on windows and linux

2. Is it an error or a hang or a crash?

error

3. What error(s) or behavior you are seeing?

DPY-2004: "parameters" argument should be a list of sequences or dictionaries, or an integer specifying the number of times to execute the statement

4. Does your application call init_oracle_client()?
does not work regardless if it is thin or thick mode.

5. Include a runnable Python script that shows the problem.
```
import pyodbc
cnxn = pyodbc.connect('DRIVER={SQL Server};SERVER=SQLSRV01;DATABASE=DATABASE;UID=USER;PWD=PASSWORD')
cursor = cnxn.cursor()
cursor.execute("SELECT 1, 2 UNION SELECT 3, 4")
results = cursor.fetchall()

with oracledb.connect(user=un, password=pw, dsn=cs) as connection:
    with connection.cursor() as oracle_cursor:
        oracle_cursor.executemany("insert into ParentTable values (:1, :2)", results)
```
I get the following error on the last line:
```DPY-2004: "parameters" argument should be a list of sequences or dictionaries, or an integer specifying the number of times to execute the statement```

more info:
```
type(results) -- returns <class 'list'>
type(results[0]) -- return <class 'pyodbc.Row'>
```

This works with no problem in cx_oracle!

**Comment by hl1** [comment]
workaround for those encountering similar issues is to convert pyodbc rows into regular list:
```
results = [list(row) for row in cursor.fetchall())]
```

but I am not looking for a workaround, I need a solution for this as pyodbc.rows are more powerful than a regular list. and converting a huge stack of code to the workaround above should be temporary and not the final solution.

**Comment by anthony-tuininga** [comment]
I have pushed a patch that should correct this issue. If you are able to [build from source](https://python-oracledb.readthedocs.io/en/latest/user_guide/installation.html#installing-from-source-code) you can verify that it corrects your issue as well.

**Comment by hl1** [comment]
> I have pushed a patch that should correct this issue. If you are able to [build from source](https://python-oracledb.readthedocs.io/en/latest/user_guide/installation.html#installing-from-source-code) you can verify that it corrects your issue as well.

I am not able to build from source but will be waiting on this to be released in future oracledb version

**Comment by anthony-tuininga** [comment]
This has been included in python-oracledb 1.4.0 which was just released.
